mrblib/haconiwa/runner.rb

```diff
@@ -74,19 +79,20 @@ def run(init_command)
         [:SIGTERM, :SIGINT, :SIGHUP, :SIGPIPE].each do |sig|
           Signal.trap(sig) do |signo|
             unless base.cleaned
-              puts "Supervisor received unintended kill. Cleanup..."
+              Logger.warning "Supervisor received unintended kill. Cleanup..."
```

Comment:
If the supervisor haconiwa process is killed unintendedly, this will be logged!

```
Nov  8 07:47:30 udzura haconiwa.haconiwa-2fff2fe7[27166]: Supervisor received unintended kill. Cleanup...
```